save-backend/src/main/kotlin/org/cqfn/save/backend/SaveApplication.kt

```diff
@@ -19,11 +14,7 @@ typealias EmptyResponse = ResponseEntity<Void>
  * An entrypoint for spring for save-backend
  */
 @SpringBootApplication
-@EnableWebFlux
```

Comment:
And this will be configured even without `@Import(ApplicationConfiguration)`? Or why we don't need configuration for application itself?

Reply:
Yes, all configuration classes are auto-discovered